### PR TITLE
Add comma-separation for Interacting_taxon_ID; add definition for Taxon_ID; add definition for Relation_ID

### PR DESCRIPTION
**Pull request by balhoff**
Some edits for #3031.

### CODE REVIEW

File: specs/gpad-gpi-2-0.md [hunk 1]
```diff
@@ -196,16 +197,22 @@ Each annotation is on a separate line of tab separated values:
  5 | Reference ::= [ID] ('\|' ID)* | | 1 or greater | PMID:30695063 | Different IDs, e.g. PMID and MOD paper id, must correspond to the same publication or reference
  6 | Evidence_type ::= OBO_ID | Evidence and Conclusion Ontology | 1 | ECO:0000315 |  Mapping file in progress:  https://github.com/evidenceontology/evidenceontology/issues/249
  7 | With_or_From ::= [ID] ('\|' \| ‘,’ ID)* | | 0 or greater | WB:WBVar00000510 | Pipe-separated entries represent independent evidence; comma-separated entries represent grouped evidence, e.g. two of three genes in a triply mutant organism
- 8 | Interacting_taxon_ID ::= NCBITaxon:[Taxon_ID] | | 0 or greater | NCBITaxon:5476 |
+ 8 | Interacting_taxon_ID ::= ['NCBITaxon:'Taxon_ID] (',' 'NCBITaxon:'[Taxon_ID])* | | 0 or greater | NCBITaxon:5476 |
```
Comment (kltm) [inline specs/gpad-gpi-2-0.md:200]: It that `| |`? maybe `| '|'`?

**Comment by kltm** [comment]
@balhoff This look fine to me (although see my comment as I may have missed something).
Do you want to add more, or can I merge this as-is?

File: specs/gpad-gpi-2-0.md (hunk 1 above)
Comment (balhoff) [reply specs/gpad-gpi-2-0.md:200]: Ah you're right! Should be comma or pipe (https://github.com/geneontology/go-annotation/issues/3031#issuecomment-641351411). I'll update.

**Comment by balhoff** [comment]
I'll make that one fix, then we can merge. I wanted to fix a little at a time.

File: specs/gpad-gpi-2-0.md (hunk 1 above)
Comment (balhoff) [reply specs/gpad-gpi-2-0.md]: @kltm I updated to match how it is done in With_or_From.

**Comment by kltm** [comment]
Delta further conversation about the "alpha" part of alphanumeric (see utf-8 convo elsewhere), I'm fine if you want to self-merge when ready.

**Pull request merged**